### PR TITLE
Add notes for 2.1.0 and 6.1.0 Chocolatey products

### PR DESCRIPTION
**Pull request by gep13**
## Description Of Changes

Add release notes for releases of 4 of the main Chocolatey products.

## Motivation and Context

There are releases coming up, and we need to get this documented.

## Testing

* [x] I have previewed these changes using the [Docker Container](https://github.com/chocolatey/docs/tree/master/.devcontainer) or another method before submitting this pull request.

## Change Types Made

* [ ] Minor documentation fix (typos etc.).
* [x] Major documentation change (refactoring, reformatting or adding documentation to existing page).
* [ ] New documentation page added.

## Change Checklist

* [ ] Requires a change to menu structure (top or left hand side)/
* [ ] Menu structure has been updated

## Related Issue

N/A

### CODE REVIEW

**Comment by gep13** [comment]
@pauby this is in draft while we finalize a few things, but I thought it would make sense to get these release notes created, since we know roughly what is going out in these releases.

File: input/en-us/agent/release-notes.md [hunk 1]
```diff
@@ -31,6 +31,21 @@ This covers the release notes for the Chocolatey Agent Service (`chocolatey-agen
 <?! Include "../../shared/chocolatey-component-dependencies.txt" /?>
 
 
+## 2.1.0 (TBD)
+
+> :choco-warning: **WARNING**
+>
+> Refer to our [Upgrade Guide](xref:upgrading-to-chocolatey-v2-v6) for recommendations before upgrading from 1.x versions to 2.0.0.
```
Comment (pauby) [inline input/en-us/agent/release-notes.md:38]:
```suggestion
> Refer to our [Upgrade Guide](xref:upgrading-to-chocolatey-v2-v6) for recommendations before upgrading from 1.x versions to 2.x.
```

Comment (pauby) [reply input/en-us/agent/release-notes.md:38]: That documentation will need to be updated to say 2x now that we're releasing 2.1.0.

File: input/en-us/choco/release-notes.md [hunk 2]
```diff
@@ -21,6 +21,33 @@ This covers changes for the "chocolatey" and "chocolatey.lib" packages, which ar
 <?! Include "../../shared/chocolatey-component-dependencies-link.txt" /?>
 
 
+## 2.1.0 (TBD)
+
+> :choco-warning: **WARNING**
+>
+> Refer to our [Upgrade Guide](xref:upgrading-to-chocolatey-v2-v6) for recommendations before upgrading from 1.x versions to 2.0.0.
```
Comment (pauby) [inline input/en-us/choco/release-notes.md:28]:
```suggestion
> Refer to our [Upgrade Guide](xref:upgrading-to-chocolatey-v2-v6) for recommendations before upgrading from 1.x versions to 2.x.
```

File: input/en-us/chocolatey-gui/release-notes.md [hunk 3]
```diff
@@ -19,6 +19,22 @@ This covers changes for the "chocolateygui" package, which is available as FOSS.
 <?! Include "../../shared/chocolatey-component-dependencies.txt" /?>
 
 
+## 2.1.0 (TBD)
+
+> :choco-warning: **WARNING**
+>
+> Refer to our [Upgrade Guide](xref:upgrading-to-chocolatey-v2-v6) for recommendations before upgrading from 1.x versions to 2.0.0.
```
Comment (pauby) [inline input/en-us/chocolatey-gui/release-notes.md:26]:
```suggestion
> Refer to our [Upgrade Guide](xref:upgrading-to-chocolatey-v2-v6) for recommendations before upgrading from 1.x versions to 2.x.
```

File: input/en-us/licensed-extension/release-notes.md [hunk 4]
```diff
@@ -43,6 +43,23 @@ Please see [Install the Licensed Edition](xref:setup-licensed) for information o
 <?! Include "../../shared/chocolatey-component-dependencies-link.txt" /?>
 
 
+## 6.1.0 (TBD)
+
+> :choco-warning: **WARNING**
+>
+> Refer to our [Upgrade Guide](xref:upgrading-to-chocolatey-v2-v6) for recommendations before upgrading from 5.x and earlier versions to 6.0.0.
```
Comment (pauby) [inline input/en-us/licensed-extension/release-notes.md:50]:
```suggestion
> Refer to our [Upgrade Guide](xref:upgrading-to-chocolatey-v2-v6) for recommendations before upgrading from 5.x and earlier versions to 6.x.
```

**Review by pauby — CHANGES_REQUESTED** [review]

**Comment by gep13** [comment]
@pauby I have made the suggested changes, and this is now ready for another review.

**Review by pauby — APPROVED** [review]

File: input/en-us/licensed-extension/release-notes.md [hunk 5]
```diff
@@ -43,6 +43,24 @@ Please see [Install the Licensed Edition](xref:setup-licensed) for information o
...
+### Bug Fixes
+
+- Fix - Package version normalization is not being applied everywhere.
+- Fix - `choco download` incorrectly uses the cached nupkg.
+- Fix - Attempting to reinstall an embedded package, a package that actually contains the application installers, can result in failure.
+- Fix - Invalid URI Format warning shown when running Chocolatey CLI command using some Chocolatey Licensed Extension command line options.
+  
+### Improvement
+
+- Performance degradation after upgrading to version Chocolatey Licensed Extension v6.0.0.
```
Comment (pauby) [inline input/en-us/licensed-extension/release-notes.md:61]:
```suggestion
- Degraded performance after upgrading to version Chocolatey Licensed Extension v6.0.0.
```

**Review by pauby — CHANGES_REQUESTED** [review]

**Comment by gep13** [comment]
@pauby I have made the suggestion. Can you do a final review and approval? Thanks!

**Review by pauby — DISMISSED** [review]

**Pull request merged**